modules/layers/src/scatterplot-layer/scatterplot-layer.ts

```diff
@@ -254,8 +256,8 @@ export default class ScatterplotLayer<DataT = any, ExtraPropsT extends {} = {}>
     const model = this.state.model!;
 
     model.setUniforms(uniforms);
-    model.setUniforms({
-      stroked: stroked ? 1 : 0,
+    const scatterplot: ScatterplotSettings = {
```

Comment:
We should standardize on Settings vs Props for uniform inputs. I have gravitated towards `...Props` in recent shadertools refactorings, but the question could still be considered open.

Reply:
There are several things in this file all named `scatterplot`. I would spell them out for clarity:
- The shader module:
    `modules: [project32, picking, scatterplotUniforms]`
- The module props:
    `model.shaderInputs.setProps({scatterplot: scatterplotSettings})`